src/CustomerProfile.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = ({ id }) => {
+  const [profile, setProfile] = useState({});
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${id}`)
+      .then(res => res.json())
+      .then(data => {
+        setProfile(data);
+      });
```

Comment:
What happens if this request goes wrong? Always good practice to handle the errors 😄